### PR TITLE
Update Mattermost entries for Go and JavaScript

### PR DESCRIPTION
**Pull request by jasonblais**
Thanks for a great repository! 

Mattermost recently updated labels and so I'm submitting a PR to update the queries accordingly.

If any questions, let me know!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [Hugo](https://github.com/spf13/hugo/labels/exp%2Fbeginner) _(label: exp/beginner)_ <br> A Fast and Flexible Static Site Generator built with love in GoLang
 - [CockroachDB](https://github.com/cockroachdb/cockroach/labels/E-easy) _(label: E-easy)_ <br> The Scalable, Survivable, Strongly-Consistent SQL Database
 - [Helm](https://github.com/kubernetes/helm/labels/good%20first%20issue) _(label: good first issue)_ <br> The Kubernetes Package Manager
-- [Mattermost](https://github.com/mattermost/mattermost-server/issues?q=is%3Aopen+label%3A%22Up+For+Grabs%22+label%3A%22Difficulty%3A+Introductory%22+label%3AGo) _(label: Up For Grabs, Difficulty: Introductory, Go)_ <br> Open source Slack-alternative in Golang and React
+- [Mattermost](https://github.com/mattermost/mattermost-server/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3A%22Up+For+Grabs%22+label%3A%22Difficulty%2F1%3AEasy%22+label%3A%22Tech%2FGo%22+) _(label: Up For Grabs, Difficulty/1:Easy, Tech/Go)_ <br> Open source Slack-alternative in Golang and React
```
Comment (vvscode) [inline README.md]: in fact, this is 3 labels

why just `label:"Difficulty/1:Easy"` not enough?

File: README.md [hunk 2]
```diff
@@ -134,7 +134,7 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [Fastify](https://github.com/fastify/fastify/labels/good%20first%20issue) _(label: good first issue)_ <br> Fast and low overhead web framework, for Node.js.
 - [Node.js core](https://github.com/nodejs/node/labels/good%20first%20issue) _(label: good first issue)_ <br> JavaScript runtime built on Chrome's V8 JavaScript engine
 - [Jest](https://github.com/facebook/jest/labels/good%20first%20issue) _(label: good first issue)_ <br> A complete and easy to set up JavaScript testing solution.
-- [Mattermost](https://github.com/mattermost/mattermost-server/issues?q=is%3Aopen+label%3A%22Up+For+Grabs%22+label%3A%22Difficulty%3A+Introductory%22+label%3AReactJS) _(label: Up For Grabs, Difficulty: Introductory, ReactJS)_ <br> Open source Slack-alternative in Golang and React
+- [Mattermost](https://github.com/mattermost/mattermost-server/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3A%22Up+For+Grabs%22+label%3A%22Difficulty%2F1%3AEasy%22+label%3A%22Tech%2FJavaScript%22+) _(label: Up For Grabs, Difficulty:1/Easy, Tech/JavaScript)_ <br> Open source Slack-alternative in Golang and React
```
Comment (vvscode) [inline README.md]: same question

File: README.md (hunk 1 above)
Comment (jasonblais) [reply README.md]: @vvscode good question! There are three labels for

1. `Difficulty/1:Easy` for beginner/introductory issues
2. `Up For Grabs` for open issues not already worked on by someone
3. `Tech/Go` for Go tickets; `Tech/JavaScript` for JavaScript tickets

**Comment by MunGell** [comment]
Hi @jasonblais 

Thanks for the update

**Pull request merged**